homeassistant/config.py

```diff
@@ -272,6 +272,35 @@ def _raise_issue_if_no_country(hass: HomeAssistant, country: str | None) -> None
     )
 
 
+def _raise_issue_if_legacy_templates(
+    hass: HomeAssistant, legacy_templates: bool | None
+) -> None:
+    # legacy_templates can have the following values:
+    # - None: Using default value (False) -> Delete repair issues
```

Comment:
There's no code that deletes the repair issues. Maybe we should delete it since the user can reload the core config and `async_process_ha_core_config` will be called again then?

Reply:
I added it back. I was thinking that the core yaml part can be only reloaded by a restart